libcloud/compute/drivers/dimensiondata.py

```diff
@@ -176,8 +181,8 @@ def reboot_node(self, node):
             'server/rebootServer',
             method='POST',
             data=ET.tostring(request_elm)).object
-        result = findtext(body, 'responseCode', TYPES_URN)
-        return result == 'IN_PROGRESS'
+        responseCode = findtext(body, 'responseCode', TYPES_URN)
+        return responseCode == 'IN_PROGRESS' or responseCode == 'OK'
```

Comment:
It seems like the same check is repeated all over the driver so one thing you could do is define a module / class level constant for response codes which indicate success and a utility method which performs this check, e.g. `return self._operation_successful(response_object)`.